backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -28,18 +30,18 @@
   // I'll have you know that I didn't actually mean to do this...
```

Comment:
I am adding middle names so they can be used for search matching.

Reply:
we will still want to validate against null names and single initials. hard for me to parse all the tests but please make sure the filter doesn't break against a null middle name, for example!